react/features/conference/components/native/Conference.js

```diff
@@ -484,6 +530,14 @@ function _mapStateToProps(state) {
          */
         _locationURL: locationURL,
 
+        /**
+         * Set to {@code true} when the notifications are to be displayed.
+         *
+         * @private
+         * @type {boolean}
+         */
+        _notificationsVisible: areNotificationsVisible(state),
```

Comment:
I'd really love to have this inside the NotificationContainer, and make it render null if there is nothing to render. This way we avoid mixing up conference and notification code in this class

Reply:
There are already `isFilmstripVisible` and `shouldDisplayTileView` used here in very similar fashion that's why I did it.

The `areNotificationsVisible` method takes into account more factors than simply the number of notifications. Btw. it is indeed that the container will not render itself if there are 0 notifications. It's only external factors that are taken into account in this higher level method.

Reply:
I will still move it to the `NotificationsContainer` if you insist, just wanted to mention those things.

Reply:
Also in my opinion this method `areNotificationsVisible` should probably be part of the conference feature, because it is in fact the conference code which manages which things are displayed and when based on input taken from multiple features.

Reply:
This way we would avoid interdependencies between features.

Reply:
This is what I mean: https://sketchboard.me/bBvrmSaEQKHS#/

Reply:
Going into details some of the checks in this method should be moved to the `NotificationsComponent` and the other which depends on another features should end up in the conference feature. That's my proposal.